eduroam.md

```diff
@@ -40,98 +53,190 @@ The process of connecting to the network may be different depending on the Linux
+3. In the same directory, create a configuration file named `eduroam.8021x`. In it, input the following information:
+```
+[Security]
+EAP-Method=TLS
+EAP-Identity=anonymous@rit.edu
+EAP-TLS-CACert=/var/lib/iwd/ritCACert
+EAP-TLS-ClientCert=/var/lib/iwd/encryptedCertKey.pem
+EAP-TLS-ClientKey=/var/lib/iwd/encryptedCertKey.pem
+EAP-TLS-ServerDomainMask=radius.rit.edu
+EAP-TLS-Phase2-Method=Tunneled-PAP
+EAP-TLS-Phase2-Identity=abc1234@rit.edu
+EAP-TLS-Phase2-Password=P@ssw0rd1
+
+[Settings]
+AutoConnect=true
+```
+Replace `abc1234@rit.edu` and `P@ssw0rd1` with your RIT email and password. *Do not change the `EAP-Identity` line*.
+
+4. Run the following command as root: 
+```
+# iwctl station wlan0 connect eduroam
+```
+
+5. The above command will prompt you for the password *to the `.pem` file you downloaded*. Enter that now.
+
+6. (Optional) Run the following to ensure that your connection is working: `ping 9.9.9.9`
 
-2. Once you have these two files, exit your text editor and open the folder where the files created in step 1 are in a terminal. Currently, both the certificate and key are encrypted, and while `wpa_supplicant` can work with them in their current state, it's not guaranteed. To ensure that they can be used properly, run the following commands. This will create two new files, which are decrypted copies of the certificate and key.
-```bash
-sudo openssl pkey -in personalKey.pem -out decryptedKey.pem
-sudo openssl x509 -in personalCert.pem -out decryptedCert.pem
+For any devices having issues, see [this link](https://wiki.archlinux.org/title/Iwd#Verbose_TLS_debugging) to start the debugging process. Common problems include:
+
+- Entering an incorrect location for the certificates, or entering an invalid name for the certificates. This results in a `Not configured` on attempted connection.
+- Entering `TTLS` instead of `TLS`. This results in a `journalctl` error shown below:
+```
+EAP server tried method 13 while client was configured for method 21
+EAP completed with eapFail
+4-Way handshake failed for ifindex 4, reason: 23
 ```
-The first of these commands will ask for a password. Enter the password of the file.
 
-3. Now that these files are decrypted, we can create the configuration file for `wpa_supplicant`. Open your text editor of choice and create a file in `/etc/wpa_supplicant`, and fill in the file according to the block below. You can name the file whatever you wish, but for automation purposes, its recommended to name the file `wpa_supplicant-[interface name].conf`. This makes it easier for `systemd` and `dhcpcd` to interface with `wpa_supplicant` once it's set up.
+### wpa_supplicant Configuration
+
+If you do not have a `.p12` file from RIT before starting this section, go back to the beginning of this page, and create a new certificate, downloading the `P12` option instead.
+
+This tutorial assumes that all commands are either run directly as root.
```

Comment:
this seems like it might be duplicated. also the sentence feels incomplete

Reply:
Line 192 shown above is slightly repetitive, will edit to fix.

Reply:
Even if its not repetitive, all commands now contain a `#` or `$` to indicate where commands need to be done, so its not particularly necessary anymore.